app/bundles/CoreBundle/Helper/Dsn/Dsn.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Mautic\CoreBundle\Helper\Dsn;
+
+class Dsn
```

Comment:
New classes should be final by default unless there is good reason why not. Please update that for all new classes.
```suggestion
final class Dsn
```

Reply:
Done